src/test/test_hs_service.c

```diff
@@ -1296,18 +1295,11 @@ test_service_event(void *arg)
      * descriptor map so we can retry it. */
     ip = helper_create_service_ip();
     service_intro_point_add(service->desc_current->intro_points.map, ip);
-    ip->circuit_established = 1;  /* We'll test that, it MUST be 0 after. */
-    run_housekeeping_event(now);
-    tt_int_op(digest256map_size(service->desc_current->intro_points.map),
-              OP_EQ, 1);
-    /* No removal if we have an established circuit after retries. */
```

Comment:
Hmm, these two lines try to test the #31652 mechanism. Did the patch remove the "No removal if we have an established circuit after retries" part of the test?

Reply:
I don't think so.  From the commit message:

> This commit also removes a unit test that was testing specifically that flag
> but now we rely solely on the HS circuit map which is also tested few lines
> below the removed test.

I meant by that is this is removed because just after that removed test, we test the same thing but with a registered circuit. In other words, we bumped the retry value with a registered circuit and we still have it after housekeeping.